setup.py

```diff
@@ -23,24 +23,24 @@
     packages=find_packages(exclude=("tests",)),
     include_package_data=True,
     install_requires=[
-        "cachetools>=4.1.1",
+        "cachetools>=5.3.3",
         "mistune<2.0.0",  # v2 is totally incompatible with unclear benefit
-        "requests>=2.25.0",
-        "python-json-logger>=2.0.1",
-        "Flask>=2.1.1",
+        "requests>=2.32.0",
+        "python-json-logger>=2.0.7",
+        "Flask>=3.0.0",
         "gunicorn>=20.0.0",
         "ordered-set>=4.1.0",
-        "Jinja2>=2.11.3",
-        "statsd>=3.3.0",
+        "Jinja2>=3.1.4",
+        "statsd>=4.0.1",
         "Flask-Redis>=0.4.0",
-        "pyyaml>=5.3.1",
+        "pyyaml>=6.0.1",
         "phonenumbers>=8.13.18",
-        "pytz>=2020.4",
+        "pytz>=2024.1",
         "smartypants>=2.0.1",
-        "pypdf>=3.9.0",
-        "itsdangerous>=1.1.0",
-        "govuk-bank-holidays>=0.10,<1.0",
-        "boto3>=1.19.4",
+        "pypdf>=3.13.0",
+        "itsdangerous>=2.1.2",
+        "govuk-bank-holidays>=0.14,<1.0",
```

Comment:
Optional, but I don’t think we should restrict the major version here unless there’s a known problem with `1.x` (in this case `0.14` is the highest version and there is no `1.x branch)
```suggestion
        "govuk-bank-holidays>=0.14",
```

Reply:
Done